File: Karthik/5-9_to_6-9/palindrome.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        s=0
        temp=x
        while(x>0):
            r=x%10
            s=s*10+r
            x//=10
        print(s)
        if s==temp:
            return True
        else:
            return False

File: Chandra Kiran/05-09-2022/palindrome.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function (x) {
    let num = x;
    let temp = 0;
    let i = 1;
    while (num > 0) {
        temp = temp*10 + (num) % 10 ;
        num = Math.floor(num/10);

        i = i * 10;
    }
    return (temp === x);

    // var str = "" + x;
    // var rev = ""

    // for (let i = str.length-1; i >= 0; i--){
    //     rev+= str[i]
    // }
    // return rev == x;
    
};

console.log(isPalindrome(121))
console.log(isPalindrome(10))